### DISCUSSION TITLE
browser.maximizeWindow( ) not working anymore on v7

### DISCUSSION THREAD

**Discussion by tswiftma** [webdriverio/webdriverio#7489] [Technical Discussion]
Hi I was using browser.maximizeWindow( ) in my test project but it stopped working with the following error "Error: Command "maximizeWindow" is not yet implemented". I'm using selenium-standalone to run tests in Chrome and Edge and tried every option possible to maximize the browser in my wdio.config.js file but nothing is working.

I found a comment on StackOverflow about the same issue on v6: https://stackoverflow.com/questions/63854911/error-command-maximizewindow-is-not-yet-implemented-using-webdriverio-v6:

_maximizeWindow was removed when chromedriver switched from json wire to W3C protocol and reimplemented a couple of versions ago._

Is there a workaround or another method I could call to maximize the browsers? Thanks!

**Comment by christian-bromann** [comment]
@tswiftma the command is implemented but you are using the DevTools protocol where this command does in fact not work. Make sure your Selenium standalone server is running correctly.

**Reply by tswiftma** [reply]
Thanks @christian-bromann I was able to work around it by specifying the following edge driver options in the config

wdioConf.config,
  {
    capabilities: [
      // more capabilities defined here
      // or override capabilities from wdio.conf.js
      {
        maxInstances: 1,
        browserName: 'edge',
        'ms:edgeOptions': {
          args: ['--start-maximized']
        },
        acceptInsecureCerts: true,
        //
        // Edge launches via devtools/puppeteer.
        // Need to set defaultViewport to null to use the whole browser
        'wdio:devtoolsOptions': {
          defaultViewport: null
        }
      }
    ],

**Reply by christian-bromann** [reply]
@tswiftma you are not using Edgedriver. You are running WebdriverIO with Puppeteer. If you want to use `maximizeWindow` you need to start Edgedriver, e.g. using [wdio-edgedriver-service](https://www.npmjs.com/package/wdio-edgedriver-service)
